WebRoot/skins/_base/base4/skin.properties

```diff
@@ -1601,6 +1601,16 @@ CreateContactGroupDialogTable   = @FullWidth@
 CreateContactGroupDialogCell    = padding:0 0 8px 0;
 CreateContactGroupDialogInput   = @FullWidth@
 
+#######################################################################
+#   Grant Rights Dialog
+#######################################################################
+
+GrantRightsDialogOuterContainer     = background-color:@lighten(SecondaryBrandingColor,94)@;
+GrantRightsDialogInputContainer     = width:290px; border-width:1px; @cssShadow@:inset 0 0 0.5px 0 @IconMsgStatus@; @borderRadius@:2px;
```

Comment:
IconMsgStatus has value of rgba(0, 0, 0, 0.5), which is same as that is used here.
So used that rather than creating a new one.